docs/scos/dev/feature-integration-guides/202200.0/glue-api/glue-json-api-convention-integration-guide.md

```diff
@@ -0,0 +1,502 @@
+     */
+    public function getRelationshipResourceType(): string
+    {
+        return static::RESOURCE_TYPE_DUMMY_RELATIONSHIPS;
+    }
+}
+```
+
+### 5) Set up behavior
+Enable the following behaviors by registering the plugins:
+
+| PLUGIN                                                | SPECIFICATION                                                                   | NAMESPACE                                                                                                   |
+|-------------------------------------------------------|---------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|
+| HttpCommunicationProtocolPlugin                       | Defines applicability, accepts the request and sends the response.              | Spryker\Glue\GlueHttp\Plugin\GlueApplication                                                                |
+| CorsHeaderExistenceRequestAfterRoutingValidatorPlugin | Executes validations that need to be aware of the resolved route.               | Spryker\Glue\GlueHttp\Plugin\GlueStorefrontApiApplication                                                   |
+| ConventionResourceFilterPlugin                        | Filters resources for JSON API convention.                                      | Spryker\Glue\GlueApplication\Plugin\GlueApplication                                                         |
+| ResourceRouteMatcherPlugin                            | Routes API requests using the `ResourceInterface`.                              | Spryker\Glue\GlueApplication\Plugin\GlueApplication                                                         |
+| RequestResourceFilterPlugin                           | Filters resources by `GlueRequestTransfer` resource name.                       | Spryker\Glue\GlueApplication\Plugin\GlueApplication                                                         |
+| JsonApiApiConventionPlugin                            | Allows the convention to follow the request flow.                               | Spryker\Glue\GlueJsonApiConvention\Plugin\GlueApplication                                                   |
+| SparseFieldRequestBuilderPlugin                       | Extends `GlueRequestTransfer.sparseResources` with `GlueSparseResourceTransfer` | Spryker\Glue\GlueJsonApiConvention\Plugin\GlueJsonApiConvention                                             |
+| AttributesRequestBuilderPlugin                        | Extends `GlueRequestTransfer.attributes` with content attributes.               | Spryker\Glue\GlueJsonApiConvention\Plugin\GlueJsonApiConvention                                             |
+| RelationshipRequestBuilderPlugin                      | Extends `GlueRequestTransfer.included` with included fields.                    | Spryker\Glue\GlueJsonApiConvention\Plugin\GlueJsonApiConvention                                             |
```

Comment:
Builds the `GlueRequestTransfer.includedRelationships` from the `?include=resource-type` request parameters.